pages/blog/posts/json-schema-ambassadors.md

```diff
@@ -0,0 +1,96 @@
+
+Despite JSON Schema's presence, many still see it as part of another standard. By promoting JSON Schema through various channels, we enhance its recognition and usage. This increased visibility can attract more sponsorships, funding further programs like the Ambassadors Program and fostering a better ecosystem. It's a positive cycle; once we start, momentum will help us grow.
+
+As the project thrives, so too does our community. The Ambassadors Program brings several benefits specifically for our community.
+
+"The community is very friendly, welcoming, and extremely smart. I already learnt a LOT by participating in it. I want the JSON Schema ecosystem to grow, as it will likely power my business as a result. The Ambassadors Program is a great opportunity to recognise key contributors and encourage them to continue being involved." - Juan Cruz Viotti - Founder and CTO at [Intelligence.AI](https://intelligence.ai) - Candidate Ambassador
+
+### As the Community
+
+The project and the community both benefit from the Ambassadors Program. Ambassadors will contribute valuable content, enhancing learning and engagement. To keep their ambassador status each year, ambassadors will be expected to make a number of contributions that benefit the community, such as articles, videos, podcasts, or even talking or running a workshop at a conference.
+
+Given the prominence of the ambassadors, we will, collectively, check any and all published work they produce, making sure it's accurate. If you work with code, you've probably seen your fair share of poorly documented projects, and you know the impact that can have. High-quality, up-to-date content makes it easier for both newcomers and veterans to learn and stay current.
+
+Further, we don't require or even expect that all the content ambassadors produce will go onto the project's website. No. There's actually improved reach by ambassadors using their existing channels to reach their community, hopefully expanding the JSON Schema community in addition to educating other spaces.
+
+## A portrait of an Ambassador
+
+We've long planned the Ambassadors Program and have identified initial candidates. (You can nominate yourself or others!) Here are the qualities we seek in ambassadors.
+
+Firstly, an ambassador has to be with the community. This means in the slack server, on Github, and occasionally on calls or other places representing or being a recognised expert or source of knowledge for JSON Schema.
+
+Critically, we expect ambassadors to be aligned with the JSON Schema project's mission and values, be respectful of our code of conduct, and continue to be active in the community.
+
+## Recognising our Ambassadors from the community
+
+We easily identified several community members to invite into the Ambassadors Program. The TSC and full time staff spend a LOT of time in our community spaces, so even if we don't always reply or comment, we do notice those who are active and helpful.
```

Comment:
> full time staff

This makes it sound like JSON Schema is the employer.  Maybe "full time members"?

Reply:
I agree. There is no employer and there is no staff, just contributors, some of them lucky to be paid to work full time in the project.

```suggestion
We easily identified several community members to invite into the Ambassadors Program. The TSC and the Community Team spend a LOT of time in our community spaces, so even if we don't always reply or comment, we do notice those who are active and helpful.
```